### PR TITLE
Licensing: update Manual Configuration

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 1]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
...
-Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-license.ts`. Paste the license key you copied from the Download Manager:
+---
+##### jQuery
 
-    <!-- tab: devextreme-license.ts -->export const licenseKey = 'DEVELOPER_LICENSE_KEY’;
+#### Scripts
+
+Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-key.js`. Specify your DevExtreme license key in [GlobalConfig](/api-reference/50%20Common/Object%20Structures/GlobalConfig '/Documentation/ApiReference/Common/Object_Structures/GlobalConfig'):
+
+    <!-- tab: devextreme-key.js -->DevExpress.config({ licenseKey: 'DEVELOPER_LICENSE_KEY' });
```
Comment (helen-dikareva) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]: file name where user sets the `licenseKey` property should be named `devextreme-license.js`

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 2]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
...
+
+To allow each developer to use their own license key, do not store this file in your repository. Instruct Git to ignore the file that holds the key. To do this, add the file path to your project's `.gitignore` file:
+
+    <!-- tab: .gitignore -->src/devextreme-key.js
```
Comment (helen-dikareva) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]: `devextreme-license.js` as well then

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 3]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
...
+
+This action also ensures that your team does not commit the key by accident.
+
+If your project includes sources via `<script>` tags (does not use bundlers), add a reference to the file that registers the license key: 
+
+    <!-- tab: devextreme-license.js --><script src="./dx.all.js" type="text/javascript"> </script> 
+    <script src="./devextreme-key.js" type="text/javascript"></script>
```
Comment (helen-dikareva) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]: `devextreme-license.js` as well then

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 4]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
 
 ### Manual Configuration
 
-After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration.
+After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration. Below, you will find instructions on how to set up a license key depending on whether you connect sources via `<script>` or use modules and bundlers.
```
Comment (IlyaKhD) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]:
```suggestion
After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration. Below, you will find instructions on how to set up a license key depending on whether your project is "modular" or "non modular". The difference comes down to how new files are added to the project - by using `import` (`require`) statements or by directly referencing the file from a page by using `<script>` tag.
```

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 5]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
...
 
-Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-license.ts`. Paste the license key you copied from the Download Manager:
+---
+##### jQuery
 
-    <!-- tab: devextreme-license.ts -->export const licenseKey = 'DEVELOPER_LICENSE_KEY’;
+#### Scripts
+
+Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-key.js`. Specify your DevExtreme license key in [GlobalConfig](/api-reference/50%20Common/Object%20Structures/GlobalConfig '/Documentation/ApiReference/Common/Object_Structures/GlobalConfig'):
```
Comment (IlyaKhD) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]:
```suggestion
Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-license.js`. Specify your DevExtreme license key in [GlobalConfig](/api-reference/50%20Common/Object%20Structures/GlobalConfig '/Documentation/ApiReference/Common/Object_Structures/GlobalConfig'):
```

Comment (IlyaKhD) [reply concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]: If I'm not mistaken, all occurrences of `devextreme-key.js` should be repalced with `devextreme-license.js`

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 6]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
...
 
-After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration.
+After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration. Below, you will find instructions on how to set up a license key depending on whether you connect sources via `<script>` or use modules and bundlers.
 
-Create a new file in the folder where you store your project sources. For example, your file path may look like this: `src/devextreme-license.ts`. Paste the license key you copied from the Download Manager:
+---
+##### jQuery
 
-    <!-- tab: devextreme-license.ts -->export const licenseKey = 'DEVELOPER_LICENSE_KEY’;
+#### Scripts
```
Comment (IlyaKhD) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]:
```suggestion
#### NonModular
```

I suggest using 'Modular' and 'Non Modular' terms

In the end, modules are also bundled and added to a page with `<script>` tag.

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 7]
```diff
@@ -10,11 +10,130 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
...
+    <!-- tab: .gitignore -->src/devextreme-key.js
+
+This action also ensures that your team does not commit the key by accident.
+
+If your project includes sources via `<script>` tags (does not use bundlers), add a reference to the file that registers the license key: 
+
+    <!-- tab: devextreme-license.js --><script src="./dx.all.js" type="text/javascript"> </script> 
+    <script src="./devextreme-key.js" type="text/javascript"></script>
+
+#### Modules
```
Comment (IlyaKhD) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]:
```suggestion
#### Modular
```

**Review by IlyaKhD — APPROVED** [review]

**Review by helen-dikareva — APPROVED** [review]

File: concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md [hunk 8]
```diff
@@ -10,15 +10,134 @@ Log in to the [DevExpress Download Manager](https://www.devexpress.com/ClientCen
 
 ### Manual Configuration
 
-After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration.
+After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration. Below, you will find instructions on setting up a license key based on whether your project is *modular* or *non-modular*. The distinction lies in how new files are added to the project: either through `import` (`require`) statements or by directly referencing the file from a page using `<script>` tags.
```
Comment (dirk-pieterse) [inline concepts/Common/Licensing/10 How to Obtain and Validate Your License Key.md]:
```suggestion
After you obtain the key from the DevExpress Download Manager, define that key as a constant in a separate file and use that constant in your application configuration. Below there are instructions on how to set up a license key based on whether your project is *modular* or *non-modular*. The difference is how new files are added to the project: either through `import` (`require`) statements or by directly referencing the file from a page using `<script>` tags.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**